doc/modules/lda_qda.rst

```diff
@@ -22,44 +21,92 @@ Also there are no parameters to tune for these algorithms.
 
 .. centered:: |ldaqda|
 
-The plot shows decision boundaries for LDA and QDA. The bottom row
-demonstrates that LDA can only learn linear boundaries, while QDA can learn
+The plot shows decision boundaries for LDA and QDA. The first row shows that,
+when the classes covariances are the same, LDA and QDA yield the same result 
+(up to a small difference resulting from the implementation). The bottom row demonstrates that in general, 
+LDA can only learn linear boundaries, while QDA can learn
 quadratic boundaries and is therefore more flexible.
 
 .. topic:: Examples:
 
     :ref:`example_classification_plot_lda_qda.py`: Comparison of LDA and QDA on synthetic data.
 
-
 Dimensionality reduction using LDA
 ==================================
 
-:class:`lda.LDA` can be used to perform supervised dimensionality reduction by
-projecting the input data to a subspace consisting of the most
-discriminant directions.
+:class:`lda.LDA` can be used to perform supervised dimensionality reduction, by
+projecting the input data to a linear subspace consisting of the directions which maximize the
+separation between classes (in a precise sense discussed in the mathematics section below). 
+The dimension of the output is necessarily less that the number of classes, 
+so this is a in general a rather strong dimensionality reduction, and only makes senses 
+in a multiclass setting.
+
 This is implemented in :func:`lda.LDA.transform`. The desired
 dimensionality can be set using the ``n_components`` constructor
 parameter. This parameter has no influence on :func:`lda.LDA.fit` or :func:`lda.LDA.predict`.
 
+.. topic:: Examples:
+
+    :ref:`example_decomposition_plot_pca_vs_lda.py`: Comparison of LDA and PCA for dimensionality reduction of the Iris dataset
+
+Mathematical formulation of the LDA and QDA classifiers
+======================
```

Comment:
Interesting, I don't see an error either. Could you still please extend the underline to be at least as long as the title?